scheduler/plugin/plugin.go

```diff
@@ -406,7 +407,13 @@ func (pl *wasmPlugin) PostBind(ctx context.Context, state *framework.CycleState,
 	}
 
 	defer pl.pool.freeFromBinding(pod.UID) // the cycle is over, put it back into the pool.
-	// TODO: partially implemented for testing
+	params := &stack{pod: pod, nodeName: nodeName}
+	ctx = context.WithValue(ctx, stackKey{}, params)
+	if err := pl.pool.doWithSchedulingGuest(ctx, pod.UID, func(g *guest) {
+		g.postBind(ctx)
+	}); err != nil {
+		klog.Error(err)
```

Comment:
Please use contextual logging, not directly use klog.Hoge.

Reply:
@sanposhiho ,
Thanks for your comment.
Just to clarify, which library should I use?

Reply:
klog has the contextual logging feature.
like https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/framework/plugins/dynamicresources/dynamicresources.go#L174

Reply:
@sanposhiho 
Thanks! Fixed it, PTAL